paramak/parametric_components/toroidal_field_coil_princeton_d.py

```diff
@@ -70,6 +73,14 @@ def azimuth_placement_angle(self):
     def azimuth_placement_angle(self, value):
         self._azimuth_placement_angle = value
 
+    @property
+    def rotation_angle(self):
+        return self._rotation_angle
+
+    @rotation_angle.setter
+    def rotation_angle(self, value):
+        self._rotation_angle = value
+
```

Comment:
This is maybe not needed since the rotation_angle setters and getters are already defined as the RotateMixedSHape level